pandas/io/formats/style.py

```diff
@@ -2,7 +2,11 @@
 Module for applying conditional formatting to
 DataFrames and Series.
 """
-from collections import MutableMapping, defaultdict
+try:
```

Comment:
you can just change this to use ``is_dict_like`` instead I think

Reply:
I am not following here. `is_dict_like` instead of ??

Reply:
Got it. Done. It is much cleaner.